src/index.js

```diff
@@ -1,17 +1,20 @@
-import Main from './pages/Main/Main';
+import { router } from './router/Router';
+import urls from './router/urls';
+import routes from './router/routes';
 
-// routing
+let root = document.getElementById('root');
 
-let sessionCookie
+if (root === null) {
+  root = document.createElement('div', { id: 'root' });
+}
 
-document.addEventListener('DOMContentLoaded', function() {
-    sessionCookie = document.cookie.split('; ').find(row => row.startsWith('session='));
-    let root = document.getElementById('root');
-  
-    if (root === null) {
-      root = document.createElement('div', { id: 'root' });
-    }
-    new Main(root, sessionCookie).render();
-});
+let startingPath;
+router.register(routes);
 
+if (window.location.pathname === '/') {
+  startingPath = urls.base;
+} else {
+  startingPath = window.location.pathname;
+}
 
+router.go(startingPath);
```

Comment:
Кажется, что теперь можно просто оставить так:
```suggestion
router.go(window.location.pathname);
```

Reply:
fixed